backend/Watchdog.Core/Watchdog.Core.BLL/Services/IssueService.cs

```diff
@@ -139,22 +153,61 @@ private async Task<ICollection<IssueMessage>> GetIssueMessagesAsync()
         private async Task<int> GetTotalHitsAsync<T>() where T : class
         {
             var totalHits = await _client.CountAsync<T>();
-            return (int) totalHits.Count;
+            return (int)totalHits.Count;
         }
 
-        public async Task UpdateAssignee(UpdateAssigneeDto assigneeDto)
+        public Task UpdateAssigneeAsync(UpdateAssigneeDto assigneeDto)
         {
-            var issueResponse = await _client.GetAsync<Issue>(assigneeDto.IssueId);
-            if (!issueResponse.IsValid)
+            if (assigneeDto is null)
+            {
+                throw new System.ArgumentNullException(nameof(assigneeDto));
+            }
+
+            return UpdateAssigneeInternalAsync(assigneeDto);
+
+        }
+        private async Task UpdateAssigneeInternalAsync(UpdateAssigneeDto assigneeDto)
```

Comment:
Add enter after the end bracket, remove enter after return

Reply:
Done